cmd/nodemon/nodemon.go

```diff
@@ -154,10 +155,20 @@ func run() error {
 		return err
 	}
 
-	notifications := scraper.Start(ctx)
+	privateNodes, err := ns.Nodes(true) // get private nodes aka specific nodes
+	if err != nil {
+		zap.Error("failed to get specific nodes", zapLogger.Error(err))
+		return err
+	}
+	initialTS := time.Now().Unix()
+	initialPrivateNodesEvents := make([]entities.EventProducerWithTimestamp, len(privateNodes))
+	for i, node := range privateNodes {
+		initialPrivateNodesEvents[i] = entities.NewUnreachableEvent(node.URL, initialTS)
```

Comment:
I think we should move the whole initialization to `storing/private_nodes/private_nodes.go`

Reply:
Yep, I thought about it. Okay, will fix

Reply:
Fixed